flutter/simple_chat/lib/chat_room/chat_room_page.dart

```diff
@@ -0,0 +1,217 @@
+import 'dart:async';
+
+import 'package:djangoflow_auth/djangoflow_auth.dart';
+import 'package:djangoflow_chat/djangoflow_chat.dart';
+import 'package:djangoflow_openapi/djangoflow_openapi.dart';
+import 'package:djangoflow_websocket/djangoflow_websocket.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_chat_ui/flutter_chat_ui.dart' as chat_ui;
+import 'package:flutter_chat_types/flutter_chat_types.dart' as types;
+import 'package:simple_chat/api_repository.dart';
+import 'package:jwt_decode/jwt_decode.dart';
+import 'package:simple_chat/simple_chat_routes.dart';
+
+class ChatRoomPage extends StatelessWidget {
+  const ChatRoomPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final (roomId, name) =
+        ModalRoute.of(context)!.settings.arguments as (int, String);
+
+    final authCubit = context.read<AuthCubit>();
+    final myId = authCubit.state.token != null
+        ? Jwt.parseJwt(authCubit.state.token!)['user_id'] ?? 0
```

Comment:
@oguibueno Cool idea! I liked your approach. 

For fetching user details we also have `AuthUsersDataBloc`. I think I forgot to mention about it in the docs. It accepts id: `0` as filter to return current user's UserIdentity object, which also contains actual user id, names etc.